x-pack/plugins/cloud_security_posture/server/saved_objects/migrations/csp_rule_template.ts

```diff
@@ -19,14 +20,15 @@ function migrateCspRuleMetadata(
   doc: SavedObjectUnsanitizedDoc<CspRuleTemplateV830>,
   context: SavedObjectMigrationContext
 ): SavedObjectUnsanitizedDoc<CspRuleTemplateV840> {
-  const { enabled, muted, ...metadata } = doc.attributes;
+  const { enabled, muted, benchmark, ...metadata } = doc.attributes;
   return {
     ...doc,
     attributes: {
       enabled,
       muted,
       metadata: {
         ...metadata,
+        benchmark: { ...benchmark, id: CIS_INTEGRATION_INPUTS_MAP['cloudbeat/vanilla'] },
```

Comment:
migrating rule **template** `V830` to `V840` now also includes assigning the default benchmark id `cis_k8s` as all existing rules are from that benchmark